server_cmds.go

```diff
@@ -15,129 +15,149 @@ type ServerMethods struct {
+	TemplateServerDefaultGroup     int    `ms:"serverinstance_template_serverdefault_group"`
+	TemplateChannelAdminGroup      int    `ms:"serverinstance_template_channeladmin_group"`
+	TemplateChannelDefaultGroup    int    `ms:"serverinstance_template_channeldefault_group"`
+	PermissionsVersion             int    `ms:"serverinstance_permissions_version"`
+	PendingConnectionsPerIP        int    `ms:"serverinstance_pending_connections_per_ip"`
+	ServerQueryMaxConnectionsPerIP int    `ms:"serverinstance_serverquery_max_connections_per_ip"`
 }
 
 // ServerConnectionInfo represents the connection info for a TeamSpeak 3 instance.
+// hostinfo
 type ServerConnectionInfo struct {
-	FileTransferBandwidthSent     uint64  `ms:"connection_filetransfer_bandwidth_sent"`
-	FileTransferBandwidthReceived uint64  `ms:"connection_filetransfer_bandwidth_received"`
-	FileTransferTotalSent         uint64  `ms:"connection_filetransfer_bytes_sent_total"`
-	FileTransferTotalReceived     uint64  `ms:"connection_filetransfer_bytes_received_total"`
-	PacketsSentTotal              uint64  `ms:"connection_packets_sent_total"`
-	PacketsReceivedTotal          uint64  `ms:"connection_packets_received_total"`
-	BytesSentTotal                uint64  `ms:"connection_bytes_sent_total"`
-	BytesReceivedTotal            uint64  `ms:"connection_bytes_received_total"`
-	BandwidthSentLastSecond       uint64  `ms:"connection_bandwidth_sent_last_second_total"`
-	BandwidthReceivedLastSecond   uint64  `ms:"connection_bandwidth_received_last_second_total"`
-	BandwidthSentLastMinute       uint64  `ms:"connection_bandwidth_sent_last_minute_total"`
-	BandwidthReceivedLastMinute   uint64  `ms:"connection_bandwidth_received_last_minute_total"`
-	ConnectedTime                 uint32  `ms:"connection_connected_time"`
-	PacketLossTotalAvg            float32 `ms:"connection_packetloss_total"`
-	PingTotalAvg                  float32 `ms:"connection_ping"`
-	PacketsSentSpeech             uint64  `ms:"connection_packets_sent_speech"`
-	PacketsReceivedSpeech         uint64  `ms:"connection_packets_received_speech"`
-	BytesSentSpeech               uint64  `ms:"connection_bytes_sent_speech"`
-	BytesReceivedSpeech           uint64  `ms:"connection_bytes_received_speech"`
-	PacketsSentKeepalive          uint64  `ms:"connection_packets_sent_keepalive"`
-	PacketsReceivedKeepalive      uint64  `ms:"connection_packets_received_keepalive"`
-	BytesSentKeepalive            uint64  `ms:"connection_bytes_sent_keepalive"`
-	BytesReceivedKeepalive        uint64  `ms:"connection_bytes_received_keepalive"`
-	PacketsSentControl            uint64  `ms:"connection_packets_sent_control"`
-	PacketsReceivedControl        uint64  `ms:"connection_packets_received_control"`
-	BytesSentControl              uint64  `ms:"connection_bytes_sent_control"`
-	BytesReceivedControl          uint64  `ms:"connection_bytes_received_control"`
+	InstanceUptime                    uint64 `ms:"instance_uptime"`
```

Comment:
Would be nice to have this in as time.Duration and time.Time

Reply:
should be resolved with my latest commit.